pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -133,7 +133,8 @@ func (nm *NodeManager) removeNode(uuid string, node *v1.Node) {
 	if name != "" {
 		delete(nm.nodeNameMap, name)
 	} else {
-		klog.V(4).Info("node name: ", node.GetName(), " has a different uuid. Skip deleting this node from cache.")
+		klog.V(4).Info("node name: ", node.GetName(), " has a different uuid. Delete this node from cache.")
```

Comment:
```suggestion
		klog.V(4).Info("node name: ", node.GetName(), " has a different uuid. Delete this node from cache. This could happen if VM is rebooted, and SystemUUID changes")
```